people.yaml

```diff
@@ -1606,16 +1606,19 @@ rahul-clarice:
      agreement: institution
      institution: Clarice
      email: rahul.varma@claricetechnologies.com
+     expires_on: 2015-03-16
 jotiram:
      name: Jotiram Bhagat
      agreement: institution
      institution: Clarice
      email: jotiram.bhagat@claricetechnologies.com
+     expires_on: 2015-03-16
 abhra-clarice:
      name: Abhradeep Mazumder
      agreement: institution
      institution: Clarice
      email: abhradeep.mazumder@claricetechnologies.com
+     expires_on: 2015-03-16
 swapnali-clarice:
```

Comment:
I don't believe so. We're still working with Clarice on Android stuff for a little while longer and this was the list I was given to remove.